sdk/react/src/lib/fragment-components/a-notification.tsx

```diff
@@ -0,0 +1,302 @@
+import React from 'react';
+import { AComponent, ComponentInfo } from './a-component';
+import image from './../assets/component-icons/notification.svg';
+import { css, cx } from 'emotion';
+import {
+	Dropdown,
+	Checkbox,
+	TextInput,
+	ToastNotification,
+	InlineNotification
+} from '@carbon/react';
+import {
+	angularClassNamesFromComponentObj,
+	nameStringToVariableString,
+	reactClassNamesFromComponentObj
+} from '../helpers/tools';
+
+const preventCheckEventStyle = css`
+	pointer-events: none;
+`;
+
+export const ANotificationSettingsUI = ({ selectedComponent, setComponent }: any) => {
+	const kind = [
+		{ id: 'error', text: 'Error' },
+		{ id: 'info', text: 'Info' },
+		{ id: 'info-square', text: 'Info square' },
+		{ id: 'success', text: 'Success' },
+		{ id: 'warning', text: 'Warning' },
+		{ id: 'warning-alt', text: 'Warning alt' }
+	];
+	const varients = [
```

Comment:
```suggestion
	const variants = [
```

but also questioning if this is necessary or should it be multiple components. At the moment leaning to multiple since that's how they are implemented in both frameworks plus they are used for different things in design.

_If not, ids could still be simpler (not using `Notification`)_